src/data_reader.py

```diff
@@ -39,7 +43,11 @@ def load_data(self, data_type: DataType):
             data_path = self.proofs_data_path
 
         payload = DataReader.create_image_payload(data_path)
+        import time
```

Comment:
Is timing still needed here?

Reply:
Oh naah, need to add better logging, sometimes the request took a lot of time for the original images, so had added that.